src/pug/blog/report_22.pug

```diff
@@ -0,0 +1,49 @@
+extends _blog.pug
+
+block append subtitle
+  | Progress report #22
+
+block content
+  h2 Progress report #22
+  +news_header('12 April 2024', 'report_21', '')
+
+  h3 Wireless Alpakka (the odyssey continues)
+  p In the last report we explained our criteria to evaluate #[em Pico-w + BT] (with and without dongle) and how we were not able to achieve acceptable results.
+
+  p During the last weeks we focused on evaluating #[em Pico-w + WLAN], and while the initial results on latency (~2 ms) and jitter were very satisfactory, we found critical issues with #[+ax('https://github.com/georgerobotics/cyw43-driver/issues/111') connection dropping] that we were no able to solve either, which also makes the solution unsuitable.
+
+  p Ultimately we have to ask ourselves if BT or WLAN are the #[em right tool] for low-latency videogame peripherals, and the answer is possibly "no".
+
+  p So even though we may go back to one of these solutions if the blockers are solved, for now we decided to explore alternatives without the Pico-W:
+
+  ul
+    li Keep the original Pico (or bare #[em RP2040]), but with a #[em standalone RF module] best suited for our usecase.
```

Comment:
- Combine the current Pico with another RF module (on a re-designed Alpakka PCB)
- Re-design the Alpakka PCB, integrating an RP2040, as well as a suitable RF module
- Re-design the Alpakka PCB, integrating a suitable Espressif ESP32 SoC (System on Chip)
- Re-design the Alpakka PCB, integrating a suitable Nordic Semiconductors SoC